src/estimators/any_sketch.py

```diff
@@ -47,6 +47,40 @@ def __call__(self, x, y):
     return x | y
 
 
+class UniqueKeyFunction(ValueFunction):
+  """ValueFunction to track the state of unique key of a register."""
+
+  FLAG_EMPTY_REGISTER = -1
```

Comment:
Can this be zero? It simplifies a lot of code below if it can.

Reply:
Good point. I was setting it as -1 because our ID (from set_generator) can be 0. 

In this version, I just set FLAG_EMPTY_REGISTER as 0, so we don't need to change any_sketch.